tests/device.cc

```diff
@@ -64,6 +68,9 @@ BOOST_AUTO_TEST_CASE (unit_test_device)
   DevicePtr_t robot;
   LiegroupSpacePtr_t space;
 
+  robot = Device::create ("robot");
+  BOOST_CHECK(pinocchio::checkData (robot->model(), robot->data()));
```

Comment:
This check needs https://github.com/stack-of-tasks/pinocchio/pull/1012/commits/9cd8d2ccd6262fd3b459a28ca1eb2fd7499478d3 to succeed. It is currently only on the devel branch.

Reply:
Even with the version of eigen3 that comes with 18.04 ? (3.3.4)

Reply:
No, you are right. That's only true for Eigen <= 3.2.92.